### PR TITLE
Login added

### LINKED ISSUE #9: Login

### PR DESCRIPTION
**Pull request by jgrgurovic**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -22,6 +22,6 @@
       "@/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "views/homepage/OrganizationsSections"],
```
Comment (ilovric00) [inline tsconfig.json]: why this?? :O

**Review by ilovric00 — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (jgrgurovic) [reply tsconfig.json]: I don't even have and idea how it got there or why, but I'll remove it now

File: src/app/login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+              <p className="text-sm font-light text-gray-500 dark:text-gray-400">
+                Do not have an account yet?
+                <a
+                  href="#"
+                  className="font-medium text-primary-600 hover:underline dark:text-primary-500"
+                >
+                  Sign up
+                </a>
+              </p>
+            </form>
```
Comment (ilovric00) [inline src/app/login/page.tsx:153]: I would like to see `react-hook-form` here

**Review by ilovric00 — APPROVED** [review]
Smooth 👏

File: src/app/login/page.tsx (hunk 2 above)
Comment (ilovric00) [reply src/app/login/page.tsx:153]: You shouldn't merge this PR without fixing these things

**Comment by ilovric00** [comment]
You shouldn't merge this PR without fixing these things

File: src/app/login/page.tsx (hunk 2 above)
Comment (jgrgurovic) [reply src/app/login/page.tsx:153]: The issue in question was fixed just in a different PR as the file incorrectly ended up here as a result of a ' branch mixup'. The form can be seen fixed in the PR 'Updated homepage' under [c6d0523] and the issue in tsconfig.json was fixed here [6a1a823]. If I overlook comments by mistake and if there is another comment I missed here feel free to tell me :)

**Pull request merged**